### DISCUSSION TITLE
Read message from UI but got 400 Bad Request

### DISCUSSION THREAD

**Discussion by Zeeeeta** [provectus/kafka-ui#3277] [Q&A] [answered]
I have a single node Kafka (kafka_2.13-3.3.2.tgz) cluster running on a Linux server (no internet).
It is a default setup using KRaft and the only config I have set is the /config/kraft/server.properties: `advertised.listeners=PLAINTEXT://192.168.20.109:9092`, where 192.168.20.109 is the LAN IP.

I am using UI for Apache Kafka v0.5.0 (027d9b4) running locally on my PC on port 8080, directly from Java without using docker.
application-local.yml as below:

```
kafka:
  clusters:
    - name: "[UAT] Kafka"
      bootstrapServers: 192.168.20.109:9092
```
Only port 9092-9093 firewall rules opened between my PC and the server. 

I created a test topic using the UI, and produced some test message.
![image](https://user-images.githubusercontent.com/26813698/215379510-96b7e6d4-a3a7-4546-853a-44ca73ab9318.png)

However, when I try to view the message using the UI, it still shows no messages found and when I check the API call, the response is 400 Bad Request.
![image](https://user-images.githubusercontent.com/26813698/215379531-de061a55-d7ae-447e-b703-c38bbcdee8d8.png)

May I get some help on what the problem could be?
Thanks.

**Comment by Haarolean** [comment, marked as answer]
Hi, can you verify that it works fine with any other cluster without special symbols (`[`,`]`,` `)?

**Comment by Zeeeeta** [comment]
Oh it worked, thanks for the help.

**Reply by Haarolean** [reply]
Glad it works! Probly we encode the cluster name the wrong way
